Research/ResearchUI/iOS/RSDStepViewController.swift

```diff
@@ -709,8 +709,13 @@ open class RSDStepViewController : UIViewController, RSDStepController, RSDCance
     open func shouldConfirmCancel() -> Bool {
         return self.stepViewModel.rootPathComponent.shouldConfirmCancel()
     }
-    
-    /// Finish canceling the task. This is called once the cancel is confirmed by the user.
+	
+	/// Hook to let tasks know that user decided not to cancel.
+	open func keepGoing() {
```

Comment:
Also minor: what do you think about naming this something related to canceling?  Something like `didCancelAction`?